launcher/container_runner.go

```diff
@@ -49,13 +50,15 @@ type ContainerRunner struct {
 }
 
 const (
-	// hostTokenPath defined the directory in the host that will store attestation tokens
-	hostTokenPath = "/tmp/container_launcher/"
+	// tmpContainerLauncherPath defined the directory in the host that will store attestation tokens and experiment data.
```

Comment:
please do this rename in a separate PR, ideally putting it in a const package. @jkl73 was planning on doing this.